docs/docs/building.md

```diff
@@ -27,89 +26,63 @@ contract MyContract is Migratable {
 }
 ```
 
-You might have noted that our sample contract has an `initialize` function instead of the standard contstructor. This is a requirement of ZeppelinOS's upgradeabiltiy system, you can learn more about this in our [Advanced topics](advanced.md) section. 
+Notice that our sample contract has an `initialize` function: this works as a constructor for our upgradeable contract. This is a requirement of the [ZeppelinOS's upgradeability system](advanced.md#initializers-vs-constructors).
 
-You can now compile the contract with:
+Now let's compile the contract:
 
-```
+```sh
 npx truffle compile
 ```
 
-## Initial Deployment
+## Initial deployment
 
-Register all the contract implementations of the first version of your
-project running:
+To deploy our app, we need to register the first version of our contract:
 
+```sh
+zos add MyContract
 ```
-zos add <contract_name_1>
-zos add <contract_name_2>
-...
-zos add <contract_name_n>
-```
-
-In our example, run:
 
-    zos add MyContract
+> **Note**: If you are working in a local development network like [Ganache](http://truffleframework.com/ganache/), you will need to [configure](http://truffleframework.com/docs/getting_started/project#alternative-migrating-with-ganache) your `truffle.js` file before running the `push` command.
 
-If you are working in a local development network like [Ganache](http://truffleframework.com/ganache/), you will need to [configure](http://truffleframework.com/docs/getting_started/project#alternative-migrating-with-ganache) your `truffle.js` file before running the `push` command.
+We can now push our application to the network by running:
 
-You can now push your application to the desired network by running:
-
-```
+```sh
 zos push --network <network>
 ```
 
-which will create a `zos.<network>.json` file with all the information specific to the chosen network. The format of this file is discussed in the [Advanced topics](advanced.md) section.
+This  will create a `zos.<network>.json` file with all the information specific to the chosen network. You can read more about this file in the [advanced topics](advanced.md#format-of-zosjson-and-zos-network-json-files) section.
 
-To create an upgradeable version of each of your contracts, run:
+To create an upgradeable version of our contract, we need to run:
 
-```
-zos create <contract_name_1> --network <network>
-zos create <contract_name_2> --network <network>
-...
-zos create <contract_name_n> --network <network>
+```sh
+zos create MyContract --init initialize --args 42 --network <network>
```

Comment:
While I buy into the idea of not duplicating abstract/example specific code, I'm not sure I'd keep the `<network>` parameter undetermined here.. If we make it concrete, let's go concrete all the way and use `development` (noting, of course, that other networks can be used..)

Reply:
I didn't like this either but `truffle init` from the set up guide creates an empty truffle config file.

Reply:
This will be fixed by the coming version of the cli, which will create a `development` network in the `truffle.js` file (see https://github.com/zeppelinos/zos-cli/pull/202). Please use `development`.